runway/blueprints/staticsite/staticsite.py

```diff
@@ -581,6 +594,17 @@ def _get_index_rewrite_role_function_and_version(self):
         )
         return res
 
+    def _get_code_template(self, relative_template_location):  # pylint: disable=no-self-use
```

Comment:
Can we drop this function in favor the native cfngin parameterized lookup? Should be functionally about the same (returning a Join CFN object instead that can be used where the string is used).

Reply:
Are you talking about this? https://docs.onica.com/projects/runway/en/latest/apidocs/runway.cfngin.lookups.handlers.file.html#runway.cfngin.lookups.handlers.file.parameterized_codec

Reply:
Yes

Reply:
Updated to use `read_value_from_path`